cdc/scheduler/internal/tp/agent.go

```diff
@@ -116,7 +116,7 @@ func NewAgent(ctx context.Context,
 				zap.String("namespace", changeFeedID.Namespace),
 				zap.String("changefeed", changeFeedID.ID),
 				zap.Error(err))
-			return nil, nil
+			return result, nil
```

Comment:
Is there any test cover the change?

Reply:
no, `NewAgent` is based on `etcdClient` and other stateful parameters, does not have ut yet.